modules/config.lua

```diff
@@ -45,6 +47,22 @@ config.sdl_logs_port = 6676
 config.ExitOnCrash = true
 --- Flag which defines whether ATF starts SDL on startup
 config.autorunSDL = true
+--- Security
+--
+--- Define security protocol
```

Comment:
Shouldn't this be DTLS?

Reply:
@JackLivio Previously, when this ATF feature was designed, TLS was a default. But now, DTLS is supported as well and default can be changed. Please find update in 472aa0d